src/BasketBundle/Controller/BasketController.php

```diff
@@ -468,7 +468,7 @@ public function finalReviewStepAction(Request $request)
 
         $violations = $this
             ->get('validator')
-            ->validate($basket, ['elements', 'delivery', 'payment']);
+            ->validate($basket, null, ['elements', 'delivery', 'payment']);
```

Comment:
If I understand correctly, this method has always been like this: https://github.com/symfony/symfony/commit/718601c6c3642fa7a65a408016a34feef00fe44d#diff-4666f31ca4bcde46b4880e4a8d20e584R49

So you are not only fixing compatibility with Symfony 3, and should probably target the stable branch, at least for this change, correct?

Reply:
The validator services uses the `RecursiceValidator` class, which only had a signature change in 3.0 (https://github.com/symfony/symfony/commit/2a6b629e39a3069f67af94e991c7a1b672c7b983#diff-8ca64353ad65881b21945a227035b8e0L118)
In Symfony 2.8, the validator class implements 2 interfaces, the legacy interface with the old signature, and the new interface with the new signature

Reply:
Oh ok then…